### PR TITLE
Added default Stalebot configuration.

### PR DESCRIPTION
**Pull request by markdorison**
## Description
Added default Stalebot configuration.

## Motivation / Context
Can be [inherited](https://github.com/probot/stale/blob/673afc72556932d4fb777ad724e5b97aeec5decd/.github/stale.yml#L1) (or ignored/overridden) in our other organization repos.

## Testing Instructions / How This Has Been Tested
N/A

## Documentation
See diff.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+repositories within the the [`ChromaticHQ`](https://github.com/ChromaticHQ)
+organization.
 
 **This repository is public**. It needs to be public for other repositories to
 be able to leverage the templates that it provides.
 
 The approach was taken from
 [this help page](https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file).
+
+## Stalebot Configuration
```
Comment (adamzimmermann) [inline README.md:13]:
```suggestion
## Stalebot Configuration

```
Manual markdown linting. 😂

**Review by adamzimmermann — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (adamzimmermann) [reply README.md:13]: I see we have linting here that didn't catch this, but I thought other repos were flagging this. 🤷

Comment (markdorison) [reply README.md:13]: We have yaml linting on this repo, but not automated markdown linting.

Comment (markdorison) [reply README.md:13]: 🤔 

There must be different rules in play then what I was linting with locally for markdown.

**Pull request merged**